homework_1/js/hw_1.js

```diff
@@ -0,0 +1,58 @@
+//String
+
+let string = 'some test string';
+
+// Получить первую и последнюю буквы строки
+console.log (string.length);
+console.log (string[0]);
+console.log (string[15]);
+
+// Сделать первую и последнюю буквы в верхнем регистре
+let myString = string[0].toUpperCase() + string.slice(1,15) + string[15].toUpperCase();
```

Comment:
Could use let myString = string[0].toUpperCase() + string.slice(1, string.length -1) + string[string.length -1].toUpperCase();

Reply:
Got it. Thank you for review.